my-contacts/src/ContactEnzyme.test.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import Contact from "./Contact";
+
+
+let contact = shallow(
```

Comment:
I think setting up this contact once in your test file is bad practice. If any of your tests effect the state of the component. it could break other tests. I would always set up a brand new component in each of my unit tests.  Up to you if that is done in each `it` case or if you like you can do it with a `beforeEach `

Reply:
Great call!